tests/python/pants_test/pants_run_integration_test.py

```diff
@@ -373,7 +380,8 @@ def run_pants_with_workdir(self, command, workdir, config=None, stdin_data=None,
     handle = self.run_pants_with_workdir_without_waiting(command, workdir, **kwargs)
     return handle.join(stdin_data=stdin_data, tee_output=tee_output)
 
-  def run_pants(self, command, config=None, stdin_data=None, extra_env=None, **kwargs):
+  def run_pants(self, command, config=None, stdin_data=None, extra_env=None, cleanup_workdir=True,
```

Comment:
```suggestion
  def run_pants(self, command, config=None, stdin_data=None, extra_env=None, **kwargs
```
This was a mistake before I realized that `self.pants_results()` already existed -- the `cleanup_workdir` arg can be removed entirely.